client/src/components/Survey.tsx

```diff
@@ -1,83 +1,101 @@
 import axios from 'axios';
 import React, { useState, useEffect } from 'react';
+import SurveyTag from './SurveyTag';
+import { redirect, useNavigate } from 'react-router-dom';
+import { Box, Button, Typography, Container, Grid, Paper } from '@mui/material';
```

Comment:
If redirect isn't being used it should be removed on **line 4**

Divide MUI imports into individual imports `import Box from '@mui/material/Box'` etc

Reply:
Taken out now